packages/augur-sdk/src/constants.ts

```diff
@@ -2,16 +2,56 @@ import { utils as ethersUtils } from "ethers";
 import { BigNumber } from "bignumber.js";
 
 export enum ACCOUNT_TYPES {
-U_PORT= "uPort",
-LEDGER = "ledger",
-PRIVATE_KEY = "privateKey",
-UNLOCKED_ETHEREUM_NODE = "unlockedEthereumNode",
-META_MASK = "metaMask",
-TREZOR = "trezor",
-EDGE = "edge",
-};
+  U_PORT = "uPort",
+  LEDGER = "ledger",
+  PRIVATE_KEY = "privateKey",
+  UNLOCKED_ETHEREUM_NODE = "unlockedEthereumNode",
+  META_MASK = "metaMask",
+  TREZOR = "trezor",
+  EDGE = "edge",
+}
+
+export enum SubscriptionEventNames {
```

Comment:
```suggestion
export enum SUBSCRIPTION_EVENT_NAMES {
```
We have been using uppercase underscore for enums.